src/extlibs/hrvo/agent.h

```diff
@@ -20,12 +21,12 @@ class Agent
      * @param prefVelocity       The preferred velocity of this agent.
      * @param maxSpeed           The maximum speed of this agent.
      * @param maxAccel           The maximum acceleration of this agent.
-     * @param goalIndex          The index of the Goal which this agent should go to.
+     * @param path               The path for this agent
      * @param goalRadius         The goal radius of this agent.
```

Comment:
This comment should be removed

Reply:
Remove the parameter comment in the spec?

Reply:
Im referring to goalRadius comment